install_influxdb.sh

```diff
@@ -0,0 +1,30 @@
+#/bin/bash
+set -e
+
+# Usage
+if [ -z "$1" ]; then
+    printf "\nInfluxDB version not provided.\n"
+    printf "usage: %s <INFLUXDB VERSION>\n" "$0"
+    exit 1
+fi
+
+# Download
+FILEURL="http://s3.amazonaws.com/influxdb/influxdb_$1_amd64.deb"
+TEMPFILE=$(mktemp --suffix=.deb)
+printf "Trying to download $FILEURL, saving as $TEMPFILE\n"
+curl --fail --progress-bar -L "http://s3.amazonaws.com/influxdb/influxdb_$1_amd64.deb" -o "$TEMPFILE"
```

Comment:
er... this should probably be:
`curl --fail --progress-bar -L ${FILEURL} -o "$TEMPFILE"`
?